solvers/qpSpline/PiecewisePolynomial.cpp

```diff
@@ -0,0 +1,116 @@
+#include "PiecewisePolynomial.h"
+#include <stdexcept>
+#include <cassert>
+#include <algorithm>
+
+using namespace std;
+
+PiecewisePolynomial::PiecewisePolynomial(std::vector<Polynomial> const& polynomials, std::vector<double> const& segment_times) :
```

Comment:
Yup, don't like the name PPTrajectory because of the abbreviation. If a new user wonders 'hey is there a class that does piecewise polynomial trajectories?', PiecewisePolynomial is much more likely to be found than PPTrajectory.
I would be OK with renaming to PPTrajectory if it's temporary, but I just realized another major difference: PiecewisePolynomial is 1D, whereas PPTrajectory can be N-D. Maybe we need a different class that contains a bunch of PiecewisePolynomials to match PPTrajectory, and call that PPTrajectory?